crowbar_framework/app/models/node_object.rb

```diff
@@ -1249,6 +1249,19 @@ def poweroff
     bmc_cmd("power off")
   end
 
+  def service(service_name, action)
+    platform, version = @node["platform"], @node["platform_version"]
+    case service_name
+    when :chef
+      service_cmd = ChefObject.service_command(platform, version, "chef-client", action)
+    end
+    if service_cmd.nil?
+      Rails.logger.error("service: init command for #{platform} #{version} could not be determined")
+    else
+      ssh_cmd(service_cmd)
+    end
+  end
+
```

Comment:
Can you move this a few lines later (before `disk_owner`)? Right now, it's in the middle of some BMC-related methods, which is weird.